lib/account/account-manager.js

```diff
@@ -83,6 +145,147 @@ module.exports = function (accountService, config, hotp, otDate, promise, random
 
 
     /**
+     * Creates a password hash using the same method
+     * an account holder used to create a password to login.
+     *
+     * @param {string} password
+     * @param {string} salt
+     * @return {Promise.<string>}
+     */
+    function hashPasswordAsync(accountFile, salt) {
```

Comment:
This belongs in the service.  It's not a business "feature" to make sure that the password is hashed.  It is a technical need.  Without this, the software is pointless, so that means it is in the technical realm.